src/sass/ui/_minimal.scss

```diff
@@ -0,0 +1,26 @@
+.user-interface {
+    &--minimal {
+        &.messenger {
+            header,
+            footer {
+                display: none;
```

Comment:
The reason I did it with `display: none;` instead of `ng-hide` or `ng-show` is because these other two options do exactly the same: they just add a class and hide it with `display: none;`

Reply:
Yeah, I think it's good to keep theme overrides entirely in CSS.